quotes.py

```diff
@@ -1,20 +1,69 @@
 import random
+import requests
+import logging
+import asyncio
 
-def pull_random_quote():
+QUOTE_FILE_ADDRESS = 'https://raw.githubusercontent.com/Gnomeball/QuoteBotRepo/main/quotes.txt'
 
-    with open("quotes.txt", "r") as file:
```

Comment:
This part of the patch *might* be unnecessary because you'll probably be running this with python 3.